src/ServiceControl.Audit.Persistence.RavenDB/ServerConfiguration.cs

```diff
@@ -18,6 +20,9 @@ public ServerConfiguration(string dbPath, string serverUrl, string logPath, stri
         }
 
         public string ConnectionString { get; }
+        public string ClientCertificatePath { get; internal set; }
```

Comment:
Not that it really matters. Just pointing out. Here the setters are internal and in other files they are public.